src/pages/GameBooking.js

```diff
@@ -1,278 +1,444 @@
-import React, {Component} from 'react';
+import React, {
+    Component
+} from 'react';
 import Typography from "@material-ui/core/Typography";
-import {NarrowContent, WideContent} from "../common/Content";
+import {
+    NarrowContent,
+    WideContent
+} from "../common/Content";
 import Spinner from "../common/LoadingDiv";
 import Grid from '@material-ui/core/Grid';
 import withStyles from "@material-ui/core/styles/withStyles";
-import {inject, observer} from "mobx-react";
+import {
+    inject,
+    observer
+} from "mobx-react";
 import TextField from "@material-ui/core/TextField";
 import Button from "@material-ui/core/Button";
 import classNames from 'classnames';
 import LocationIcon from '@material-ui/icons/LocationOn';
 import CalendarIcon from '@material-ui/icons/CalendarToday';
-import {dateToString, weekdayOf} from "../utils";
+import {
+    dateToString,
+    weekdayOf
+} from "../utils";
 import Autocomplete from '@material-ui/lab/Autocomplete'
 
-
+//game booking page /games/game/[session-hash]/book
```

Comment:
Not sure if this is really needed, but if you think this makes the code easier to read, maybe it would be a better idea maybe to add this in the docstring for the default component?

Reply:
Will do with next PR.